tests/common_setup.py

```diff
@@ -81,6 +81,11 @@ def class_persistent_standard_setup_one_client_bootstrapped(request):
     return standard_setup_one_client_bootstrapped_impl(request)
 
 
+@pytest.fixture(scope="function")
+def class_persistent_standard_setup_one_client_bootstrapped_rt(request):
+    return standard_setup_one_client_bootstrapped_impl(request)
```

Comment:
This is exactly `standard_setup_one_client_bootstrapped` above, which is the function scoped one.

Reply:
Same below

Reply:
I forgot to add "ignore" tag at the top. I am testing that vector since yesterday. and today Ole agreed that this can be a valid lead. thanks for the comments, but I am still unsure if that fixes the problem.

Reply:
what I want to say: ignore for now.